sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -130,6 +128,7 @@ public static void beforeClass() throws Exception {
             .withSavedCachesFolder(savedCache)
             .withShutdownHook(shutdownHook)
             .withJMXPort(jmxPort)
+            .cleanDataFilesAtStartup(false)
```

Comment:
Is it possible that these data files won’t be cleaned up at all if it’s always called with `cleanDataFilesAtStartup(false)`?

Reply:
Should not because TemporaryFolder ClassRule cleans the temp directory even if an exception is thrown.

Reply:
optionally if you want we can make the test fail if TemporaryFolder rule fails to clean but maybe risky. WDYT ?

Reply:
Well, I'm ok with current implementation if it should be deleted by default. I just asked to make sure that it won't cause useless disk usage.